app/models/support_interface/applications_filter.rb

```diff
@@ -25,6 +25,10 @@ def filter_records(application_forms)
         application_forms = application_forms.where('phase IN (?)', applied_filters[:phase])
       end
 
+      if applied_filters[:interviews]
+        application_forms = application_forms.joins(application_choices: [:interviews])
```

Comment:
Could we perhaps have a test for this one? Just a unit test for the filter. I added a similar one in #3967

Reply:
Ok, added